src/video/sdl2_opengl_v.cpp

```diff
@@ -50,7 +50,7 @@ static OGLProc GetOGLProcAddressCallback(const char *proc)
 
 bool VideoDriver_SDL_OpenGL::CreateMainWindow(uint w, uint h, uint flags)
 {
-	return this->VideoDriver_SDL_Base::CreateMainWindow(w, h, SDL_WINDOW_OPENGL);
+	return this->VideoDriver_SDL_Base::CreateMainWindow(w, h, flags | SDL_WINDOW_OPENGL);
```

Comment:
this was is funny .. `flags` is actually never used from the top layer, but only exists to have optional flags to the base layer. No caller ever actually sets any flags. Just a bit poor OO design (I think I did this? :D )